### DISCUSSION TITLE
Vs code not show auto complete for Blitz import

### DISCUSSION THREAD

**Discussion by IRediTOTO** [blitz-js/blitz#3007] [Q&A] [answered]
Hi, I am trying to use auto complete of VS code for blitz import, but it not work, I must write import manual. Do you have any idea for this ? 
![image](https://user-images.githubusercontent.com/15695252/143912008-e303a70b-73ba-4744-8a00-161983731a1a.png)

**Comment by beerose** [comment, marked as answer]
Related issue: https://github.com/blitz-js/blitz/issues/2788#issuecomment-981841215
